src/app/services/actions/query-action-creators.ts

```diff
@@ -119,15 +119,16 @@ async function createHistory(response: Response, respHeaders: any, query: IQuery
   return result;
 }
 
-export function checkTeamsAppInstallation(query: IQuery): Function {
+export function toggleRSCPopup(query: IQuery): Function {
```

Comment:
renamed this action since I felt the function name was a bit misleading: 
it not only checks if the teams app is installed, it also dispatches changes to the pop up  and now we will also have a localStorage check. ultimately the action toggles the RSC popup to appear/not appear.